code/drasil-example/Drasil/SSP/DataDefs.hs

```diff
@@ -44,28 +44,28 @@ intersliceWtrFQD = mkQuantDef watrForce intersliceWtrFEqn
 
 intersliceWtrFEqn :: Expr
 intersliceWtrFEqn = completeCase [case1,case2,case3]
-  where case1 = ((inxi slopeHght - inxi slipHght) $^ 2 / 2  *
-          sy waterWeight + (inxi waterHght - inxi slopeHght) $^ 2 *
-          sy waterWeight, inxi waterHght $>= inxi slopeHght)
+  where case1 = ((square (inxi slopeHght $- inxi slipHght) $/ dbl 2) `mulRe`
```

Comment:
`half` ?

Reply:
Thanks! Fixed

Reply:
@balacij what is the difference between `half` and an exact 2?  In the slope stability problem most of the 2s come from needing from taking moments of forces about the centre of a "slice".  They are exactly 2, because they come from mathematical definitions.  (The 3s are exact too.)